cmd/uplink/cmd/setup.go

```diff
@@ -157,9 +157,7 @@ func cmdSetup(cmd *cobra.Command, args []string) (err error) {
 	fmt.Println(`
 Your Uplink CLI is configured and ready to use!
 
-Some things to try next:
-
-* See https://documentation.tardigrade.io/api-reference/uplink-cli for some example commands`)
+* See http://documentation.tardigrade.io/api-reference/uplink-cli for some example commands`)
```

Comment:
Please, keep this URL https.

Reply:
You're right. I just copied what is written in the mockup (http).

Reply:
The mockup was based on an older state of the setup wizard. We changed the URL to https during the bakeoff week.

Reply:
still no change to `https`

Reply:
Done